docs/content/contributing/internal/RemoteCache.md

```diff
@@ -0,0 +1,42 @@
+# Remote Cache
+
+???+ warning 
+    Currently available only for Avito employees because accessible only via corporate network
+
+???+ warning
+    Persistence volume not set for now, deleting a pod will lead to losing cache.
+
+Official docs: https://docs.gradle.com/build-cache-node
+
+## How to enable locally
+
+???+ Prerequisites 
+    Ask for <GRADLE_CACHE_NODE_HOST> in internal chat
+    
+Set environment variable `GRADLE_CACHE_NODE_HOST`
+
+## How to deploy
+
+???+ Prerequisites
+    - DOCKER_REGISTRY env set
+    - GRADLE_CACHE_NODE_HOST env set
+    - [kubernetes credentials configured](http://links.k.avito.ru/kubectl)
+    - Write access to `gradle-remote-cache` namespace in cluster
```

Comment:
Is it feasible?
See comments to MBS-5444.

Reply:
mentioned this issue in doc